test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -6467,13 +6467,19 @@ def manager(self, cmd, obj_type, attrib=None, id=None, extend=None,
                        i.e. silent mode
         :type logerr: bool
         :raises: PbsManagerError
-        :returns: if expect is false: return code of qmgr
-                if expect is True: 0 if expect passes and 1 if it fails
-        """
-
-        # expect doesn't work for 'create' operations. It also doesn't
-        # work for 'unset' operations for default PBS attributes.
-        # So, only do expect automatically for 'set' and 'delete'
+        :returns: On success:
+                - if expect is False, return code of qmgr/pbs_manager
+                - if expect is True, 0 for success
+                On Error/Failure:
+                - if qmgr/pbs_manager() failed, raises PbsManagerError
+                - if expect() failed, it raises PtlExpectError
```

Comment:
I'm not sure the convention with Sphinx comments.  Should these be in a :raises: tag or here?